client/my-sites/stats/activity-log/index.jsx

```diff
@@ -100,6 +100,13 @@ class ActivityLog extends Component {
 		window.scrollTo( 0, 0 );
 	}
 
+	handlePeriodChange = ( { date, direction } ) => {
+		this.props.recordTracksEvent( 'calypso_activitylog_monthpicker_change', {
+			date,
```

Comment:
@tyxla I'm not sure what you're suggesting to change here. The passed props are strings and I don't believe moments are intended to be passed. Would you like to add `propType` validation on `date` to the `StatsPeriodNavigation` component?